apps/desktop-wallet/src/utils/app-data.ts

```diff
@@ -21,15 +21,20 @@ export const KEY_APPMETADATA = 'alephium/desktop-wallet/appmetadata'
 export const currentVersion: string = import.meta.env.VITE_VERSION
 export const isRcVersion: boolean = currentVersion.includes('-rc.')
 
-export interface AppMetaData {
-  lastTimeGitHubApiWasCalled: Date
+export interface AppMetadataGitHub {
+  lastTimeGitHubApiWasCalledForLatestVersion: Date
+  lastTimeGitHubApiWasCalledForAnnoumcenent: Date
+}
+
+export interface AppMetaData extends AppMetadataGitHub {
```

Comment:
It's kinda weird that the more generically named `AppMetadata` extends `AppMetadataGitHub` IMO. Should be do it the other way around?

```ts
export type AppMetadataGitHub = {
  lastGithubCallForLatestVersion: Date
  lastGithubCallForAnnouncement: Date
}

export type AppMetaData = {
  lastAnnouncementHashChecked: string
} & AppMetadataGitHub
```

Reply:
That's the same thing, no?

Reply:
Mhhh in my logic, you **extend** from a more generic, more global type.

In my version, it's an intersection, you don't have any hierarchy involved. Maybe it's just a matter of personal pref?

Reply:
fixed.